pkg/imgpkg/registry/registry.go

```diff
@@ -262,6 +281,10 @@ func (r *SimpleRegistry) transport(ref regname.Reference, scope string) (http.Ro
 	defer r.transportAccess.Unlock()
 	rt := r.roundTrippers.RoundTripper(ref.Context(), scope)
 	if rt == nil {
+		if r.keychain == nil {
```

Comment:
ditto, here.  We're basically saying: a) either no auth is required, or b) the credentials _and transport_ are presumed to be configured by some other mechanism (namely through the GGCR options) than what the `SimpleRegistry` typically provides.

That all true?

Reply:
Yes correct